deploy.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+echo "WARNING! USE PonyPixel AT YOUR OWN RISK!"
+echo "Usage of PonyPixel may result in restrictions placed on your Reddit account, banning you from placing tiles on r/place."
+echo "Avoid using your main Reddit account."
+echo "If you understand the risk, press enter to proceed. Ctrl+C to cancel."
+read -r temp
+echo "Testing APT (Debian and Ubuntu) and DNF (Fedora/RHEL)..."
+if [ -e "$PREFIX/bin/apt" ] ; then
+    apt install git python3 python3-pip python3-venv -y
+fi
+if [ -e "$PREFIX/bin/dnf" ] ; then
+    dnf install git python3 python3-pip
+fi
+if [ -e "$PREFIX/bin/pacman" ] ; then
+    pacman -S git
+    pacman -S python3
+    pacman -S python3-pip
+fi
+echo "Activating virtual environment..."
+python3 -m venv env
+. env/bin/activate
+echo "Installing requirements..."
+pip install -r requirements.txt
```

Comment:
Shouldn't it just be pip3 install, instead of specifying the (nearly) same command twice? As `python3-pip` is installed, running `pip3` should work.
A better alternative would be to run `python3 -m pip install -r requirements.txt` as you'll be sure to run the right pip, I think.

Reply:
No. Sometimes on Linux, running only one will strangely leave module resolving not working. I had to repeat module installation to make sure they are installed.

Reply:
I see.